pyproject.toml

```diff
@@ -6,21 +6,22 @@ authors = ["KBase Team <info@kbase.us>"]
 license = "MIT"
 
 [tool.poetry.dependencies]
-python = "^3.7"
+python = "3.9.19"
 kbase_workspace_client = "0.2.1"
 biopython = "1.77"
 docker = "4.2.2"
 PyYAML = "5.3.1"
 requests = "2.24.0"
 beautifulsoup4 = "4.9.1"
 markdown2 = "2.3.9"
-confluent-kafka = "1.5.0"
+confluent-kafka = "2.6.1"
 
 [tool.poetry.dev-dependencies]
```

Comment:
The “poetry.dev-dependencies” section is deprecated and will be removed in a future version. Use “poetry.group.dev.dependencies” instead.

Reply:
fixed